forest/src/cli/chain_cmd.rs

```diff
@@ -9,8 +9,13 @@ use crate::cli::{cli_error_and_die, handle_rpc_err};
 use cid::Cid;
 use forest_json::cid::CidJson;
 use rpc_client::chain_ops::*;
+use std::collections::HashMap;
+use strfmt::strfmt;
 use time::OffsetDateTime;
 
+const OUTPUT_PATH_DEFAULT_VALUE: &str =
```

Comment:
I'd say it's more of a `OUTPUT_PATH_DEFAULT_FORMAT`.

Reply:
Changed